src/main/java/com/movierecommender/project/MovieService.java

```diff
@@ -1,4 +1,37 @@
 package com.movierecommender.project;
 
+import org.springframework.stereotype.Service;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.io.IOException;
+
+@Service
 public class MovieService {
+
+    private static final String API_URL = "https://api.themoviedb.org/3/movie/24?language=en-US";
+    private static final String API_TOKEN = "your_api_token_here";
+
+    public String getMovie() {
+        HttpClient client = HttpClient.newHttpClient();
+        HttpRequest request = HttpRequest.newBuilder()
+                .uri(URI.create(API_URL))
+                .header("accept", "application/json")
+                .header("Authorization", "Bearer " + API_TOKEN)
+                .GET()
+                .build();
+
+        try {
+            HttpResponse<String> response = client.send(request, HttpResponse.BodyHandlers.ofString());
+            if (response.statusCode() == 200) {
+                return response.body();
+            } else {
+                return "Error: " + response.statusCode();
+            }
+        } catch (IOException | InterruptedException e) {
+            e.printStackTrace();
+            return "Error occurred while fetching movie data.";
+        }
+    }
```

Comment:
I implemented the MovieService class to fetch movie data from an external API using the HttpClient library. The getMovie method constructs a HTTP GET request that processes the response. If the API returns a successful response, it provides the movie data, otherwise, it returns an error message. This was done to enable easy integration with external movie data sources.